feat(scim): clarify that you can deprovision owners (#6253)
getsentry/sentry-docs: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx

  - Billing
  - Member
- Invalid role names will prevent group members from being provisioned. To try again, you'll need to remove the group first.

> - For security reasons, the "Owner" role cannot be provisioned through SCIM. However, you <i>can</i> deprovision users who have the "Owner" role in Sentry but are not provisioned through SCIM.

Reviewer: ```suggestion
- For security reasons, the "Owner" role cannot be provisioned through SCIM. However, you <i>can</i> deprovision users who have the "Owner" role in Sentry, but aren't provisioned through SCIM.
```
